packages/webapp/pages/plus/gift/[id].tsx

```diff
@@ -0,0 +1,85 @@
+import type { ReactElement } from 'react';
+import React from 'react';
+
+import dynamic from 'next/dynamic';
+import { useViewSize, ViewSize } from '@dailydotdev/shared/src/hooks';
+import { useRouter } from 'next/router';
+import type { GetServerSideProps } from 'next';
+import { getUserShortInfo } from '@dailydotdev/shared/src/graphql/users';
+import type { GiftUserContextData } from '@dailydotdev/shared/src/components/plus/GiftUserContext';
+import { GiftUserContext } from '@dailydotdev/shared/src/components/plus/GiftUserContext';
+import type { NextSeoProps } from 'next-seo/lib/types';
+import { getPlusLayout } from '../../../components/layouts/PlusLayout/PlusLayout';
+import { getTemplatedTitle } from '../../../components/layouts/utils';
+import { defaultOpenGraph } from '../../../next-seo';
+
+const PlusMobile = dynamic(() =>
+  import(
+    /* webpackChunkName: "plusMobile" */ '@dailydotdev/shared/src/components/plus/PlusMobile'
+  ).then((mod) => mod.PlusMobile),
+);
+const PlusDesktop = dynamic(() =>
+  import(
+    /* webpackChunkName: "plusDesktop" */ '@dailydotdev/shared/src/components/plus/PlusDesktop'
+  ).then((mod) => mod.PlusDesktop),
+);
+
+type PlusPageProps = Pick<GiftUserContextData, 'giftToUser'>;
+
+const seo: NextSeoProps = {
+  title: getTemplatedTitle('Unlock Premium Developer Features with Plus'),
+  openGraph: { ...defaultOpenGraph },
+  description:
+    'Upgrade to daily.dev Plus for an ad-free experience, custom feeds, bookmark folders, clickbait shield, and more.',
+};
+
+const PlusGiftPage = ({ giftToUser }: PlusPageProps): ReactElement => {
+  const { isReady } = useRouter();
+  const isLaptop = useViewSize(ViewSize.Laptop);
+
+  if (!isReady) {
+    return null;
+  }
+
+  return (
+    <GiftUserContext.Provider value={{ giftToUser }}>
+      {isLaptop ? <PlusDesktop /> : <PlusMobile />}
+    </GiftUserContext.Provider>
+  );
+};
+
+PlusGiftPage.getLayout = getPlusLayout;
```

Comment:
Maybe we need to add a padding top to this layout because Plus is attached to header 
<img width="950" alt="image" src="https://github.com/user-attachments/assets/1496ebed-a4e9-4e27-bd46-1a604d0b5509" />

Reply:
Makes sense. Let me add something.

Reply:
What size do you get this? I see that there is a `p-6` class on the mobile view.